metal_small/beast_settings.txt

```diff
@@ -0,0 +1,70 @@
+# """ Data Model interface v3.0
+# BEAST datamodel for the example based on M31 PHAT data
+# """
+import numpy as np
+from astropy import units
+
+# BEAST imports
+from beast.physicsmodel.stars import isochrone
+from beast.physicsmodel.stars import stellib
+from beast.physicsmodel.dust import extinction
+from beast.observationmodel.noisemodel import absflux_covmat
+
+# condensed settings just for simulations
+# (see beast-examples/phat_small for beast_settings.txt with extensive comments)
+
+project = "beast_metal_small"
+
+surveyname = "METAL"
+
+filters = ['HST_WFC3_F225W', 'HST_WFC3_F275W','HST_WFC3_F336W',
+           'HST_ACS_WFC_F475W','HST_ACS_WFC_F814W',
+           'HST_WFC3_F110W','HST_WFC3_F160W']
+
+basefilters = ["F225W", "F275W", "F336W", "F475W", "F814W", "F110W", "F160W"]
+obs_colnames = [f.upper() + "_RATE" for f in basefilters]
+
+# not needed?
+obsfile = "data/14675_LMC-13361nw-11112.gst_samp.fits"
+
+## observation(noise) model
+
+astfile = "data/14675_LMC-13361nw-11112.gst.fake.fits"
+ast_colnames = np.array(basefilters)
+noisefile = project + "/" + project + "_noisemodel.grid.hd5"
+absflux_a_matrix = absflux_covmat.hst_frac_matrix(filters)
+
+## physics grid
+
+n_subgrid = 1
+
+# stars
+
+velocity = 262.2 * units.km / units.s  # LMC velocity from SIMBAD
+distances = [18.5]
+distance_unit = units.mag
+distance_prior_model = {'name': 'flat'}
+
+logt = [6.0, 10.13, 1.0]
+age_prior_model = {'name': 'flat'}
+mass_prior_model = {"name": "kroupa"}
+z = [0.03, 0.019, 0.008, 0.004]
```

Comment:
To match the pattern of the production run settings, perhaps change this to 
```
z = (10 ** np.array([-2.1, -1.5, -0.9, -0.3]) * 0.0152).tolist()
```
which corresponds to `[0.00012074, 0.00048067, 0.00191357, 0.00761805]`.  (I cut out the highest metallicities since they wouldn't be expected in the Clouds, and also did coarser spacing.)

Reply:
(also, I have no idea why it's showing 4 lines, when this is only relevant to line 51... sorry for any confusion)

Reply:
What are the units?  In other words, what comment should I add so that it makes sense to a user?

Reply:
Hmm, good question.  From the production run example, it says it's the initial surface abundance.  The numbers in the array are log(Z) relative to solar (which are units that make more sense to me, at least).  The factor of 0.0152 was somewhere in the PARSEC documentation as being the solar abundance (at t=0).

Does that make sense?